other tasks/task8.js

```diff
@@ -1,6 +1,43 @@
 (() => {
     'use strict';
-    kintone.api(kintone.api.url('/k/v1/app/form/fields.json', true), 'GET', params, (resp) => {
-        console.log(resp);
-    })
+    kintone.events.on('app.record.create.show', (event) => {
+        console.log(event);
+        event.record.Table.value = [];
+
+        const params = {
+            app: 38,
```

Comment:
現在開いているアプリIDを設定することができます！
参考：https://developer.cybozu.io/hc/ja/articles/202166300-アプリID取得

```suggestion
            app: kintone.app.getId(),
```